quick-sight.tf

```diff
@@ -0,0 +1,72 @@
+# #########################################
+# QUICK SIGHT
+# Groups created accordingly to:
+# https://aws.amazon.com/premiumsupport/knowledge-center/quicksight-redshift-private-connection/
+# #########################################
+resource "aws_security_group" "quick_sight_sg" {
+  count       = var.enable_quick_sight ? 1 : 0
+  name        = "${module.labels.id}-quick-sight"
+  vpc_id      = module.vpc.vpc_id
+  description = lower("${module.labels.id}-quick-sight-service security group")
```

Comment:
I think we should unify the way we interpolate strings across `covid-green-infra`, in some places we use `format()` while in other places we use plain string interpolation.

I think for the purpose of this PR this is fine, but imo have it the same across all codebase would be better.